modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/api/resources/BaseExtensionResourcesGeneratorAnnotationProcessor.java

```diff
@@ -71,8 +76,11 @@
  * @since 3.7.0
  */
 @SupportedAnnotationTypes(value = {"org.mule.runtime.extension.api.annotation.Extension"})
-@SupportedSourceVersion(SourceVersion.RELEASE_8)
-@SupportedOptions(BaseExtensionResourcesGeneratorAnnotationProcessor.EXTENSION_VERSION)
+@SupportedSourceVersion(RELEASE_8)
+@SupportedOptions({
+    BaseExtensionResourcesGeneratorAnnotationProcessor.EXTENSION_VERSION,
+    BaseExtensionResourcesGeneratorAnnotationProcessor.EXTENSION_VERSION
```

Comment:
twice the same thing? do you mean resources?

Reply:
🤦🏼